cli/schemas/config-file.v1.json

```diff
@@ -111,7 +111,35 @@
           "uniqueItems": true,
           "default": ["deno.window"],
           "items": {
-            "type": "string"
+            "type": "string",
+            "examples": [
```

Comment:
Shouldn't this be `oneOf` with a string or an enum? Seems like adding a custom library will cause a diagnostic.

Reply:
I think it would be good if it could be autocompletion for now as in unstable.
https://github.com/denoland/deno/blob/2212d7d814914e43f43dfd945ee24197f50fa6fa/cli/schemas/config-file.v1.json#L561-L587

I could cover everything as an enum like lint rule, but I think it would be noisy if autocompletion is applied to `deno.shared_globals`, `es2015.collection`, and other types that most users don't use.
https://github.com/denoland/deno/blob/2212d7d814914e43f43dfd945ee24197f50fa6fa/cli/build.rs#L11-L125

Reply:
btw, I think it would be better to change to enum and add tests for unstable features, so I'll work on another PR.